module/Database/view/database/meeting/installform.phtml

```diff
@@ -120,6 +125,55 @@ $(document).ready(function () {
                 $('#install-organ').prop('disabled', true);
 
                 // enable member change events
+                $('button.reappoint').click(function(e)
```

Comment:
We should really make the JS code separate javascript files, because this causes it to be uncacheable

Reply:
Very nice prompt though

Reply:
Reappointing the chair first and then another member causes the chair function to be shown in the prompt. Clicking Yes causes that decision to be listed. Subsequently saving causes the 'real' chair to be reappointed twice as chair.

![image](https://github.com/GEWIS/gewisdb/assets/17340320/edf35226-236f-49d9-8f1a-14c38b9e135c)

![image](https://github.com/GEWIS/gewisdb/assets/17340320/478a4335-938d-474c-83c4-88f39a399cb4)

Reply:
This is a known issue for all models, I'll add the suggestion from below to all relevant code.

Reply:
> We should really make the JS code separate javascript files, because this causes it to be uncacheable

I do like that it is not cached as it always ensures that we have the latest version and do not have to do whacky stuff during deployment like we have to do with GEWISWEB. However, I do agree that the JS is a mess in that file and we should see if we can clean it up a bit (in another PR).